kanidm_tools/src/cli/account.rs

```diff
@@ -9,6 +9,10 @@ use time::OffsetDateTime;
 
 use webauthn_authenticator_rs::{u2fhid::U2FHid, WebauthnAuthenticator};
 
+use kanidm::constants::PW_MIN_LENGTH;
```

Comment:
It may be metter to move PW_MIN_LENGTH to proto for now instead? That way we dont need to include kanidm here.

Reply:
Isn't needed once the check was fixed in 35076d3